protocol/daemons/slinky/client/client_test.go

```diff
@@ -73,7 +73,7 @@ func TestClient(t *testing.T) {
 	)
```

Comment:
nit:
would be nice to add a test case to address the reason why we are making this change:
* when only 1 health check fails, it causes the node the panic

perhaps this is already covered?

Reply:
Created followup task: https://linear.app/dydx/issue/SKI-42/add-test-case-to-verify-that-price-fetcher-and-market-pair-fetcher